src/harness/testcases/testdata/exz_record_0.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
This file appears to be in the wrong location.

Reply:
Usually   all the records used in the test cases are placed in this same location right   ("src/harness/testcases/testdata/").  Maybe you meant to provide the   comment for  'src/harness/testdata/exz_record_1.json' which was placed  in a wrong location? The wrongly placed exz_record_1.json has now been removed now.

Reply:
My apologies if I mixed them up! Looks like they're all in the right place now :)